examples/getStarred.js

```diff
@@ -27,6 +19,7 @@ function getRepos (err, res) {
 }
 
 function outputStarredRepos () {
-  console.log(starredRepos.map(function (repo) { return repo['full_name'] }))
-  console.log('starred repos: ' + starredRepos.length)
+  // console.log(starredRepos.map(function (repo) { return repo['full_name'] }))
+  // console.log('starred repos: ' + starredRepos.length)
+
 }
```

Comment:
I would reduce the example to just `github.activity.getStarredRepos()` and leave pagination out it